integration-tests/multitenancy.test.ts

```diff
@@ -0,0 +1,147 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0 *
+ */
+
+import { AxiosInstance } from 'axios';
+import {
+    expectResourceToNotBePartOfSearchResults,
+    getFhirClient,
+    randomPatient,
+    waitForResourceToBeSearchable,
+} from './utils';
+import BulkExportTestHelper from './bulkExportTestHelper';
+
+jest.setTimeout(300_000);
+
+test('empty test placeholder', () => {
+    // empty test to avoid the "Your test suite must contain at least one test." error
+});
+
+if (process.env.MULTI_TENANCY_ENABLED === 'true') {
+    describe('tenant data isolation', () => {
+        let client: AxiosInstance;
+        let clientForAnotherTenant: AxiosInstance;
+        beforeAll(async () => {
+            client = await getFhirClient({ tenant: 'tenant1' });
+            clientForAnotherTenant = await getFhirClient({ tenant: 'tenant2' });
```

Comment:
In terms of set-up we created two Cognito users one with tenant1 and one with tenant2?

Reply:
yes 2 tenants are sufficient for our test cases. 

The default user for most tests (`COGNITO_USERNAME_PRACTITIONER`) is from `tenant1`. A new user (`COGNITO_USERNAME_PRACTITIONER_ANOTHER_TENANT`) is from `tenant2`